ethcore/private-tx/src/lib.rs

```diff
@@ -259,19 +272,16 @@ impl Provider where {
 
 	/// Process received private transaction
 	pub fn import_private_transaction(&self, rlp: &[u8]) -> Result<(), Error> {
-		let validator_accounts = self.config.validator_accounts.clone();
 		trace!("Private transaction received");
-		if validator_accounts.is_empty() {
-			self.broadcast_private_transaction(rlp.into());
-			return Ok(());
-		}
```

Comment:
imo this was a logic error, cause we could broadcast was done before checking the validity of private transaction rlp